### PR TITLE
add doc for persistent graph store

### LINKED ISSUE #391: Documents to describe and getting started for Store service.

### PR DESCRIPTION
**Pull request by tianliplus**
## What do these changes do?

add doc for persistent graph store

## Related issue number

Fixes #391

### CODE REVIEW

File: docs/persistent_graph_store.rst [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+After data ingested into graph storage, you need to commit data loading. The data will not be able to read until committed successfully.
+
+.. code:: bash
+
+    $ ./load_tool.sh -c commit -d hdfs://1.2.3.4:9000/tmp/data_output
+
+Notice: The later committed data will overwrite the earlier committed data which have same vertex types or edge relations.
+
+
+Running Gremlin Queries
```
Comment (siyuan0322) [inline docs/persistent_graph_store.rst:507]: I think this should be placed before realtime write, as the update or delete edges involves the queries.

File: docs/persistent_graph_store.rst [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+
+The store is a distributed graph store built on top of the popular RocksDB key value store. It adopts row-oriented design to support frequent small updates to the graph. Each row is tagged with a snapshot ID as its version. A query reads most recent version of rows relative to the snapshot ID when it starts and hence not blocked by writes. For writes we take a compromise between consistency and higher throughput. In our design writes in the same session can be grouped and executed atomically as a unit and the persistent store assigns a snapshot ID (which is a low-resolution timestamp of current time) to each group and executes groups of writes by the order of their snapshot IDs and by a deterministic (though arbitrary) order for groups of writes that occur in the same snapshot ID. It provides high write throughput while still with some degree of order and isolation though it provides less consistency than strict snapshot isolation common in database. We hope our design choice provides an interesting trade-off for practical usage.
+
+
+Known Limitation
+~~~~~~~~~~~~~~
```
[the remark this answers is not in the document]
Comment (tianliplus) [reply docs/persistent_graph_store.rst]: done

File: docs/persistent_graph_store.rst (hunk 1 above)
Comment (tianliplus) [reply docs/persistent_graph_store.rst]: done

**Review by siyuan0322 — APPROVED** [review]
LGTM.
BTW, when do you plan to make the Chinese version Document?

**Review by yecol — APPROVED** [review]

**Pull request merged**